### ISSUE TITLE
Identity values

### ISSUE THREAD

**Issue by silviacg334** [tseemann/abricate#209]
I get several identity values for the same gene in my sample, what does that mean? Are there several copies of the same gene?

**Comment by tseemann** [comment]
Yes, you probably have multiple copies of that gene in your genome,.
